### PR TITLE
EIP-1646: Decentralized Autonomous Zero-identity Protocol

### PR DESCRIPTION
**Pull request by Silkroad-zid**
A public autonomous zero-identity protocol which can greatly enhance smart contract based application's security ,for itself and for the participants, and also a core protocol to the decentralized autonomous civilization. 

Compared to existing identity related proposals the Zero-Identity has three core advantages. Firstly it is based on decentralized autonomy, meaning basically it is managed by the owner himself/herself of an Ethereum address. Secondly, It can directly interact with a smart contract and can deliver the active Ethereum address (not always itself) to the smart contract, thus it can bring the security of the smart contract's application to a high level. Thirdly, it can protect the anonymity of a Zero-Identity's owner, thus concerns about privacy disclosure would be naturally removed from Blockchain and all blockchain data could be transparent and publicly accessed. This will bring a profound change to the blockchain world.

### CODE REVIEW

**Comment by Silkroad-zid** [comment]
DAZ Smart contract: [https://github.com/Silkroad-Framework/eip](https://github.com/Silkroad-Framework/eip)

File: eip-1646.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+eip: <to be assigned>
```
Comment (axic) [inline eip-1646.md]: This needs the EIP number, which seems to be 1646 (the number of this PR).

File: eip-1646.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+title: Decentralized Autonomous Zero-identity Protocol
+author:Derek Zhou(周朝晖, zhous1998@gmail.com) (@zhous); Clinton Tong(21239506@qq.com); Yuefei Tan(whtyfhas@gmail.com)</a>; 
+discussions-to: https://github.com/ethereum/EIPs/issues/1649
+status: Draft
+type: Standards Track (ERC)
+category (*only required for Standard Track): ERC
+created: 2018-11-06
+requires (*optional): https://eips.ethereum.org/EIPS/eip-191
```
Comment (nicksavers) [inline eip-1646.md]: Just needed to mention the EIP numbers here. Not the links and not the word 'EIP'

File: eip-1646.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+A user himself/herself is the owner of his/her zero-identity. The user can revoke his/her authorization to an authorized third party. However for a minor or person who needs a guardian since he/she may not be competent to fulfill a responsibility, full access control over his/her zero-identity may be given to a third party e.g. his/her guardian while he/she may only be a user of his/her zero-identity. When he/she becomes an adult or a person that no longer needs a guardian full access control over his/her zero-identity will be transferred  to him/her with smart contracts.  
+
+## Backwards Compatibility
+There is no backwards compatibility issue. 
+
+## Implementation
+The reference implementation for ERC-1484 may be found in [Silkroad-Framework/eip](https://github.com/Silkroad-Framework/eip).
```
Comment (nicksavers) [inline eip-1646.md]: This still references ERC-1484, which seems to be a copy/paste from https://eips.ethereum.org/EIPS/eip-1484

**Pull request closed without merging**